### PR TITLE
Assistant pager for the PDA

### PR DESCRIPTION
**Pull request by BorneJim**
### Intent of your Pull Request
Adds the ability to page assistants from your PDA. Any job that is not assistant can use this and it does not require a special cartridge. 

### Why is this change good for the game?
Allows assistants to be more useful while also not filling up the common chat with your requests.
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Adds a new feautre to the PDA which allows any non-assistant to send an alert to all assistants on the station.

### What should players be aware of when it comes to the changes your PR is implementing?
Just a new feature for the PDA
### What general grouping does this PR fall under? 
PDA tweak

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
The pager has a two minute delay so it may not be spammed.

# Changelog

:cl:  
rscadd: Added a pager feature to the PDA

/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -799,13 +804,48 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 	update_icon()
 	add_overlay(icon_alert)
+/obj/item/pda/proc/receive_ping(message)
```
Comment (Bibby0110) [inline code/game/objects/items/devices/PDA/PDA.dm:808]: add new line before

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 2]
```diff
@@ -799,13 +804,48 @@ GLOBAL_LIST_EMPTY(PDAs)
...
 		return
 	send_message(U,get_viewable_pdas(), TRUE)
 
+/obj/item/pda/proc/PingAssistants (mob/living/U)
+	if (last_everyone && world.time < last_everyone + PDA_SPAM_DELAY)
+		to_chat(U,"<span class='warning'>Function is still on cooldown.</span>")
+		return
+
+	var/area/A = get_area(U)
+	var toSend = stripped_input(U, "Please enter your issue.")
```
Comment (Bibby0110) [inline code/game/objects/items/devices/PDA/PDA.dm]:
```suggestion
	var/toSend = stripped_input(U, "Please enter your issue.")
```

**Review by Bibby0110 — COMMENTED** [review]
cursed

File: code/game/objects/items/devices/PDA/PDA.dm (hunk 1 above)
Comment (adamsong) [reply code/game/objects/items/devices/PDA/PDA.dm:808]: Don't know why it was resolved but not fixed, but this needs a new line

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 3]
```diff
@@ -799,13 +804,48 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 	update_icon()
 	add_overlay(icon_alert)
+/obj/item/pda/proc/receive_ping(message)
+	if (!silent)
+		if(HAS_TRAIT(SSstation, STATION_TRAIT_PDA_GLITCHED))
+			playsound(src, pick('sound/machines/twobeep_voice1.ogg', 'sound/machines/twobeep_voice2.ogg'), 50, TRUE)
+		else
+			playsound(src, 'sound/machines/twobeep_high.ogg', 50, TRUE)
```
Comment (adamsong) [inline code/game/objects/items/devices/PDA/PDA.dm:813]: You should probably use the actual ringtone and not just the default

**Review by adamsong — CHANGES_REQUESTED** [review]
Clicked buttons two quickly, didn't properly request changes

**Review by adamsong — APPROVED** [review]
Looks good now
